ecs-cli/modules/clients/aws/ec2/mock/sdk/ec2iface_mock.go

```diff
@@ -4794,6 +4794,44 @@ func (_mr *_MockEC2APIRecorder) DescribeInstanceAttributeWithContext(arg0, arg1
 	return _mr.mock.ctrl.RecordCall(_mr.mock, "DescribeInstanceAttributeWithContext", _s...)
 }
 
+func (_m *MockEC2API) DescribeInstanceCreditSpecifications(_param0 *ec2.DescribeInstanceCreditSpecificationsInput) (*ec2.DescribeInstanceCreditSpecificationsOutput, error) {
```

Comment:
For future reference, split out changes to the mocks into a separate commit; it makes the change easier to review, since this stuff is auto-generated.

Reply:
Updates to the dependencies would also be a separate commit.

Reply:
Thanks, will commit those separately going forward.